django-verdant/rca/static/rca/js/programme-finder.js

```diff
@@ -0,0 +1,28 @@
+// Programme Finder
+$(function() {
+    $("input#course_search").autocomplete({
+        minLength: 3,
+        source: function(request, response) {
+            $.getJSON("/programme_search/?q=" + request.term, function(data) {
+                response(data);
+            });
+        },
+        select: function( event, ui ) {
+            window.location.href = ui.item.search_url || ui.item.url;
+        }
+    }).data("ui-autocomplete")._renderItem = function( ul, item ) {
+        if (item.thumbnail.url) {
+            // Render image when it's available
+            return $( "<li class=\"hero-search__result\"></li>" )
+                .data( "item.autocomplete", item )
+                .append( "<a class=\"hero-search__link\">" + "<img class=\"hero-search__image\" src=\"" + item.thumbnail.url + "\">" + item.title + "<span class=\"hero-search__link--alt\">" + item.degree_level + "</span></a>" )
```

Comment:
When searching for "Curating Contemporary Art" I get an element with that title but "NULL" written directly beneath, so I think we're missing a check for ``null`` around here somewhere